webapp/web/configuration.css

```diff
@@ -23,24 +23,28 @@
     padding: 0px;
 }
 
-.configure-package__tag-item {
+.tag-list__tag-item {
     list-style-type: none;
     border-color: #ddd;
     border-bottom-style: solid;
     border-width: 1px;
     padding: 10px;
 }
 
-.configure-package__tag-item-active {
+.tag-list__tag-item_active {
     background-color: #b5b3b3;
 }
 
-.configure-package__tag-item:hover {
+.tag-list__tag-item-drop-target {
+    background-color: #b5b3b3;
+}
+
+.tag-list__tag-item:hover {
     cursor: pointer;
     background-color: #b5b3b3;
 }
 
-.configure-package__button-add-tag-action {
+.tag-list__add-tag-action {
```

Comment:
this class and `.tag-responses__add-responses-action` look the same. It's likely we might want this button in other packages, so I suggest we can merge them into a single `.add-button` class. 

As a general suggestion - we don't need these to be explicitly part of the hierarchy, ie. we don't need to prefix them with `tag-list__` or `tag-responses__`. We can create more modular widgets/buttons/layouts that are named based on a more abstract role, not their content. That way we can also reuse them

Reply:
I agree. Refactored it.